mmv1/products/compute/ServiceAttachment.yaml

```diff
@@ -95,6 +95,10 @@ examples:
       producer_forwarding_rule_name: 'producer-forwarding-rule'
       consumer_address_name: 'psc-ilb-consumer-address'
       consumer_forwarding_rule_name: 'psc-ilb-consumer-forwarding-rule'
+  - name: 'service_attachment_cross_region_ilb'
+    primary_resource_id: 'psc_ilb_service_attachment'
+    vars:
+      name: 'tf-test'
```

Comment:
Can it be a different name? The prefix `tf-test-` is added to the name automatically. The name is `tf-test-tf-test-` in the generated provider https://github.com/modular-magician/terraform-provider-google-beta/compare/b4758b2cf0010a4c6e8426d29c74eb0c0fa97e25..f9b0ce33dbafd0aced577ac26bbf63fb8b696546#diff-9019649d6ee1a84552b947ba345402c59ef3a12112ba50dfcb46a85fdc48e420R523